test/test.go

```diff
@@ -67,26 +66,14 @@ func httpCall(apiToken, method, url string, input io.Reader, output interface{})
 		return err
 	}
 
-	b, err := httputil.DumpRequest(req, true)
```

Comment:
I removed these because the full dump seems to be unnecessarily lengthy and I don't think the it is helpful, see this build: https://app.bitrise.io/build/da875b9c-f61d-4118-a5d9-0c63dc89ef36 (Search for `(1) Test result deployment`)